wrapper.go

```diff
@@ -76,10 +74,6 @@ func New(url string) (ws *WebSocket, err error) {
 type WebSocket struct {
 	*js.Object
 
-	// Available events:
-	// open, error, close, message
```

Comment:
IMO this information is very valuable and helpful. You should preserve it.

Move it to `AddEventListener` documentation.

That way, it's a lot easier to get started with this library, since you can recognize the event types it supports and get going.